packages/web-twig/src/Resources/components/Modal/Modal.twig

```diff
@@ -0,0 +1,48 @@
+{# API #}
+{%- set _class = props.class | default(null) -%}
+{%- set _closeLabel = props.closeLabel | default('Close') -%}
+{%- set _id = props.id -%}
+
+{# Class names #}
+{%- set _bodyClassName = _spiritClassPrefix ~ 'Modal__body' -%}
```

Comment:
@crishpeen Can we define the `Modal` string only once as we do it in the `web-react` components, please?

Reply:
It would inconsistent with other components. We might fix that in the future.